### ISSUE TITLE
Does nobody have write permissions to this repo anymore?

### ISSUE THREAD

**Issue by DoNotResuscitate** [spf13/spf13-vim#1120]
Does nobody have write permissions to this repo anymore? Last update was 2016 and there are a lot of pull requests, plus a bug fix for the HTML auto tags bundle.

**Comment by Jeklah** [comment]
Yes please. I constantly run into this where it asks me for username and pass for amirh/HTML-AutoCloseTag and it is currently freezing at this point and I can't install spf13.  Anyone know a workaround?

**Comment by keithpickett** [comment]
I am just running into this as well.  The problem is that apparently `amirh/HTML-AutoCloseTag` is no longer accessible. I enter my password token from Github and it continues.  It just doesn't pull any updates from Git because it is not there.  There is apparently a fork of that repository https://github.com/heracek/HTML-AutoCloseTag as of this writing.  I will be trying it later.  For now, I will `UnBundle 'amirh/HTML-AutoCloseTag'` in my `.vimrc.bundles.local` file to get past it.  If you want to try the fork from @heracek, just add a `Bundle 'heracek/HTML-AutoCloseTag'` in the same file and re-run the `vim +BundleInstall! +BundleClean +qls` command.

Good luck!
